### PR TITLE
Add functionality to input either a username/email

### PR DESCRIPTION
**Pull request by rogerdunnhawaii**
-When logging in, you can either input a username or email

### CODE REVIEW

File: assets/scripts/auth/events.js [hunk 1]
```diff
@@ -26,9 +26,17 @@ const onSignUp = function (event) {
...
 
 const onSignIn = function (event) {
-  console.log('ran')
   event.preventDefault()
   const data = getFormFields(event.target)
+  // checking to see if input was a username or email
+  const usernameOrEmail = data.credentials.email
+  // if username is inputted, set email to null and created a username field in credentials
+  if (usernameOrEmail.search('@') === -1) {
+    console.log('did not find a @ in your string')
```
Comment (discocisco) [inline assets/scripts/auth/events.js]: Can you remove this console.log

**Comment by macitege** [comment]
Good work, thanks, Roger 🚀

**Comment by discocisco** [comment]
Good stuff

**Comment by MaxxKowalik** [comment]
looks good

**Pull request merged**